src/ng/directive/attrs.js

```diff
@@ -159,7 +159,7 @@
  *
  * @description
  *
- * This directive sets the `disabled` attribute on the element if the
+ * This directive sets the `disabled` attribute on a form element if the
```

Comment:
Yeah, I meant list all the elements that are supported *natively*, but not write that they can be used only on these elements. I'm fine with 
> (typically a form control, e.g. `input`, `button`, `select` etc.)